dev/NavigationView/NavigationView.idl

```diff
@@ -143,6 +143,7 @@ unsealed runtimeclass NavigationViewTemplateSettings : Windows.UI.Xaml.Dependenc
     Boolean SingleSelectionFollowsFocus { get; };
     Double PaneButtonWidth{ get; };
     Double LatestPaneButtonWidth{ get; };
+    Double NegativeSplitViewWidthMinusCompactLength{ get; };
```

Comment:
Is there any way we can avoid this new API ?

Reply:
I couldn't find this value in SplitView's codebehind so added this property. We can definitely overwrite the the value for that keyframeAnimation with codebehind and avoid adding this new API, but that will make re-templating difficult for some customers. 

Is there anyone familiar with SplitView animations that can offer some insight?

Reply:
@dilipojha has worked on the splitview stuff in the past

---
In reply to: [582174142](https://github.com/microsoft/microsoft-ui-xaml/pull/4328#discussion_r582174142) [](ancestors = 582174142)